src/Utilities/vector_of_constraints.jl

```diff
@@ -154,7 +154,7 @@ function MOI.get(
     v::VectorOfConstraints{F,S},
     ::MOI.ListOfConstraintIndices{F,S},
 ) where {F,S}
-    return keys(v.constraints)
+    return collect(keys(v.constraints))
```

Comment:
So I guess the issue was that we were returning the concrete list of keys, instead of a lazy evaluator?

Reply:
exactly!